### PR TITLE
Display form status on admin interface

### PR DESCRIPTION
**Pull request by remiriv**
## Proposed changes

---

Display form status for each player in admin interface

## Types of changes

---

What types of changes does your code introduce to ogre ?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Checklist

---

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

offseason_ogre/blob/master/CONTRIBUTING.md) doc

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation and modify the README.md (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If the feature changes current behavior, reasons why your solution is better.

### CODE REVIEW

File: packages/client/src/modules/administration/Games/Game/GamePlayers.tsx [hunk 1]
```diff
@@ -121,10 +128,64 @@ function buildColumns({
...
 }
 
+function buildFormColumns({
+  game,
+  teams,
+}: {
+  game: IGame;
+  teams: Team[];
+}): GridColumns<Row> {
+  const baseTeamColumn = {
```
Comment (louis-sanna-dev) [inline packages/client/src/modules/administration/Games/Game/GamePlayers.tsx]: @remiriv [detail] `baseFormColumn`

File: packages/client/src/modules/administration/Games/Game/GamePlayers.tsx [hunk 2]
```diff
@@ -121,10 +128,64 @@ function buildColumns({
...
+    flex: 1,
+    minWidth: 160,
+  } as GridColumns<Row>[0];
+  if (hasGameStarted(game.status)) {
+    return [baseTeamColumn];
+  }
+  return [
+    {
+      ...baseTeamColumn,
+      editable: true,
```
Comment (louis-sanna-dev) [inline packages/client/src/modules/administration/Games/Game/GamePlayers.tsx]: @remiriv pour que l'edit marche il faut update `processRowUpdate`

```js
        processRowUpdate={(newRow, oldRow) => {
          if (newRow.teamId !== oldRow.teamId) {
            changeTeamMutation.mutate({
              teamId: newRow.teamId,
              userId: newRow.id,
            });
          }
          return newRow;
        }}
```

**Comment by louis-sanna-dev** [comment]
@remiriv quand je double-clique sur la colonne j'ai un crash.

Screenshot 2022-11-17 at 21 13 13

 (on a plus l'effet au hover pour cette colonne)
Screenshot 2022-11-17 at 21 15 57

File: packages/client/src/modules/administration/Games/Game/GamePlayers.tsx (hunk 2 above)
Comment (remiriv) [reply packages/client/src/modules/administration/Games/Game/GamePlayers.tsx]: ah oui j'ai oublié d'enlever ça je voulais pas du tout mettre la colonne en éditable effectivement 
c'est corrigé

**Review by louis-sanna-dev — APPROVED** [review]
@remiriv good!

**Pull request merged**